src/kafkaaggregator/testtopic/agents.py

```diff
@@ -108,8 +109,9 @@ def process_window(key: Tuple, value: List[TestTopic]) -> None:
 
 
 # Tumbling window to persist test topic messages, the process_window
-# callback is called when the window expires. Window range is relative to the
-# time field in the messages.
+# callback is called when the window expires. Note that we don't have a model
+# for the test topic to specify the time field to use. An alternative is to
+# construct the window ranges relative to the timestamp added by Kafka.
```

Comment:
Will it eventually be possible to use `relative_to_field` in the future once we've got tighter integration with the source schemas from the registry? I'm assuming the aggregator needs this feature for production?

Reply:
Yes, that's something I need to investigate further, perhaps using a field descriptor with the name of the field would be enough to continue using `relative_to_field`, I'll do that as part of 
DM-24528.
--



--